app/components/phone_input_component.html.erb

```diff
@@ -51,6 +51,7 @@
         input_html: {
           class: 'phone-input__number',
         },
+        hint: tag_options[:hint],
```

Comment:
`tag_options` is ideally meant for "extra stuff" that is meant to be applied to the element but that the component otherwise doesn't have awareness of. I think it'd be clearer to use if this were included as a keyword argument of the `PhoneInputComponent#initialize` constructor.